modules/users/persons.ts

```
@@ -0,0 +1,134 @@
+import { and, eq, ilike, isNull, like, or } from 'drizzle-orm';
+
+import { db } from '@/db/drizzle';
+import { users } from '@/db/schema';
+import { isUniqueViolation } from '@/lib/db-errors';
```

ESLint

[error] There should be at least one empty line between import groups

(import/order)